projects/services-ogc/assets/owc/zkiViewer.test.context.json

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
Can you please remove this context, we can not consider any context as public.

If there are new features to test we should place them in existing contexts.

Reply:
Removed!